guides/source/active_record_migrations.md

```diff
@@ -1804,3 +1872,126 @@ Instead, consider using the
 gem provides a framework for creating and managing data migrations and other
 maintenance tasks in a way that is safe and easy to manage without interfering
 with schema migrations.
+
+### Database Views
```

Comment:
This felt like the best fot for Dtaabase Views out of the different options since Database Views are focused on how to evolve and manipulate your schema over time. Even though views are virtual and don’t store data, they are part of your schema.